apps/mobile-alphabet/src/app/app.test.ts

```diff
@@ -0,0 +1,11 @@
+import { device } from 'detox';
```

Comment:
Is this running on an actual device similar to how Cypress runs in a headless browser?